src/components/App.tsx

```diff
@@ -403,6 +405,7 @@ export class App extends React.Component<any, AppState> {
     window.addEventListener("blur", this.onBlur, false);
     window.addEventListener("dragover", this.disableEvent, false);
     window.addEventListener("drop", this.disableEvent, false);
+    this.getCanvasElement()?.addEventListener("touchstart", this.onTapStart);
```

Comment:
Let's do this inside `<canvas>` `ref` handler. This will also remove the need for `getCanvasElement`

Reply:
Oh yea. Just added it there